pennylane/templates/subroutines/qsvt.py

```diff
@@ -123,6 +124,7 @@ def qsvt(A, angles, wires, convention=None):
         A = qml.math.reshape(A, [1, 1])
 
     c, r = qml.math.shape(A)
+    global_phase, global_phase_op = None, None
```

Comment:
I had to put this in because codefactor complained that I was using global_phase before giving it any value